test/unittests.jl

```diff
@@ -93,10 +128,10 @@ end
 end
 
 @testset "reduce" begin
-    a = JACC.array([1 for i=1:10])
+    a = JACC.array([1 for i in 1:10])
```

Comment:
Maybe for later, do a formatting only PR. It seems these are not functional changes.

Reply:
Good point, I did formatting as I was going with this one, so it applied to changes from prior to this PR.